docs/pages/workflow/glossary-of-terms.md

```diff
@@ -2,18 +2,99 @@
+
+### Apple Developer Portal
+
+Apple's [official website](https://developer.apple.com/) for managing application code signing. EAS Credentials automate most of the common reasons a developer might visit this website when developing an app.
+
+### Auto Capability Signing
+
+A feature of EAS build that automatically enables or disables [Apple capabilities](#apple-capabilities) based on the project's entitlements file. [Learn more](/build-reference/ios-capabilities/).
+
+### Autolinking
+
+A cross-platform tool for automatically linking native modules to native apps via native package managers.
+
+- On iOS the tool is used in [CocoaPods](#cocoapods) `ios/Podfile` and invoked during `pod install`
+- On Android the tool is used in the `android/app/build.gradle` and invoked during the [Gradle](#gradle) sync process.
+
+There are two versions of Autolinking: Expo
+
+The default [Prebuild template](#prebuild-template) includes support for [Expo Autolinking](#expo-autolinking), and the [Community Autolinking](#community-autolinking) fork.
+
+### Babel
+
+Transpiler used for removing language features that aren't available in the runtime's JavaScript engine. [Metro](#metro-bundler) uses Babel internally. Project's can configure how Babel is used by modifying the `babel.config.js` file in their project directory. This file is optional when using [Expo CLI](#expo-cli). Expo projects should extend the default Babel preset [`babel-preset-expo`](https://github.com/expo/expo/tree/main/packages/babel-preset-expo).
+
+### CocoaPods
+
+The iOS package manager, used to link native modules to the native iOS project. This package manager is configured using the `ios/Podfile`, and updated when a user runs `pod install` in the `ios` directory.
+
+### Community Autolinking
+
+This refers to the React Native community [fork](https://github.com/react-native-community/cli/issues/248#issue-422591744) of the [Expo Autolinking](#expo-autolinking). The requirements for linking a module different to [Expo Autolinking](#expo-autolinking) but the implementation is the same.
+
+### Config Introspection
+
+A process for evaluating the results of [`expo prebuild`](#prebuild) in-memory without persisting any code changes. This is used in [auto capability signing](#auto-capability-signing) to determine what the entitlements file will look like without generating any native code. This process is also used in the [vscode expo](#vscode-expo) extension to debug [config mods](#config-mods).
+
+### Config Mods
+
+Async functions that are appended to the [Expo Config](#expo-config) for use in [Prebuild](#prebuild). This functions are given a singe native file, modify it, and return the results that should be persisted to the project. Config Mods are chained together and come from the package `@expo/config-plugins`. [Learn more](/guides/config-plugins).
```

Comment:
```suggestion
Async functions that are appended to the [Expo Config](#expo-config) for use in [Prebuild](#prebuild). These functions are given a single native file, modify it, and return the results that should be persisted to the project. Config Mods are chained together and come from the package `@expo/config-plugins`. For more information, see [Config Plugins](/guides/config-plugins).
```

Reply:
Also, the text "These functions are given a single native file, modify it", doesn't make sense. Maybe it could be: "These functions are given a single native file to modify it and ..."